wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/functions.php

```diff
@@ -81,6 +84,21 @@ function disable_parse_request( $return, $wp ) {
 }
 add_filter( 'do_parse_request', __NAMESPACE__ . '\disable_parse_request', 10, 2 );
 
+/**
+ * Get the slug of the given WP locale. This function returns a blank string if
+ * the locale is `en_US` because that is considered the default and is not
+ * prefixed in the URL paths. It also returns a blank string if `$wp_locale` is
+ * not found in the locales list.
+ */
+function get_locale_slug ( $curr_locale ) {
+	if ( $curr_locale === 'en_US' ) {
+		return '';
+	}
+
+	$locale = get_locales()[$curr_locale];
+	return isset( $locale ) ? $locale->slug : '';
```

Comment:
This is problematic as it'll cause a PHP Warning for missing `$curr_locale` key which shouldn't happen, but may.

This however can be written much easier as:
```suggestion
	return get_locales()[$curr_locale]->slug ?? '';
```

(No need to change anything here, I'm going to change it while I commit it)

Reply:
Thanks for the fix!